poetry.lock

```diff
@@ -1,41 +1,40 @@
 [[package]]
```

Comment:
It sadly appears the Poetry-lockfile-format was changed? Could this be related to https://python-poetry.org/blog/announcing-poetry-1-1-0.html? If so we should to the necessary updates to `pyproject.toml`, document the requirement of Poetry `1.1.0`, and probably version poetry-version in our Dockerfiles. We should perhaps also look into wrapping Poetry in our Makefile-setup. Note: much of what I said is beyond the scope of this PR, and should be it's own issue. 
Just check whether `poetry --version` is less than 1.1.0, and we should be good

Reply:
Yeah, seems I have `1.1.1` locally, and the Docker container had `1.1.2`.